documentation/asciidoc/computers/camera/libcamera_apps_post_processing_opencv.adoc

```diff
@@ -71,6 +71,9 @@ image::images/face_detect.jpg[Image showing faces]
 
 This stage allows text to be written into the top corner of images. It allows the same `%` substitutions as the `--info-text` parameter.
 
+Additionally to the flags of xref:libcamera_options_common.adoc#preview-window-2[`--info-text`] you can provide any token that https://www.man7.org/linux/man-pages/man3/strftime.3.html[strftime] understands to display the current date / time.
+The `--info-texttokens are interpreted first and any percentage token left is then interpreted by strftime. To achieve a datetime stamp on the video you can use e.g. `%F %T %z` (%F for the ISO-8601 date (2023-03-07), %T for 24h local time (09:57:12) and %z for the timezone difference to UTC (-0800)).
```

Comment:
`--info-texttokens` is there a missing space there?

Reply:
Yes, looks like a missing space. Also probably an extra backquote just ahead of the missing space? Thanks!

Reply:
Good catch, I'll adjust.

Reply:
Looks like you "missed" this again? :confused: